commitlint.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    extends: ["@commitlint/config-conventional"],
+    rules: {
+        "body-case": [2, "always", "sentence-case"],
+        "body-max-line-length": [2, "always", 72],
+        "header-max-length": [2, "always", 72],
+        "footer-max-length": [2, "always", 52],
+        "type-enum": [
+            2,
+            "always",
+            [
+                "chore",
+                "ci",
+                "docs",
+                "feat",
+                "fix",
+                "perf",
+                "refactor",
+                "security",
+                "style",
+                "test",
+                "release",
```

Comment:
That list does not match our naming guidelines under CONTRIBUTING.md either.
Also, I think there is no check for our convention to have #<issue-no> at the end of the first line? That would be also nice to check, if possible.